app/views/projects/show.html.erb

```diff
@@ -106,23 +106,15 @@
     <% end %>
   </div>
 
-
-
   <%= render partial: "projects/import_export" %>
 
-  <div class="btn-group">
-    <%= link_to "Edit or Delete Project", edit_project_path(@project.id), class: "button magenta"  %>
-
-    <% unless @project.parent_id %>
-      <%= link_to 'Add Sub-Project', project_new_sub_project_path(@project.id), id:"subproject", class: "button magenta" %>
-    <% end %>
+  <div class="btn-group actions">
+    <%= link_unless_archived(@project, "Edit or Delete Project", edit_project_path(@project), color: "magenta") %>
+    <%= link_unless_archived(@project, "Add Sub-Project", project_new_sub_project_path(@project), color: "magenta") unless @project.parent_id %>
     <%= link_to 'Clone Project', new_clone_project_path(@project.id), class: "button magenta" %>
     <%= link_to "#{@project.archived? ? 'Unarchive' : 'Archive'} Project", toggle_archive_project_path(@project.id), method: "patch", id: "toggle_archive", class: "button magenta", remote: true %>
-    <%= link_to "Generate Action Plan", project_action_plan_path(@project.id), class: "button" %>
-    <% if current_user.admin? %>
-      <%= link_to "View Report", project_report_path(@project.id), class: "button"  %>
-    <% end %>
+    <%= link_unless_archived(@project, "Generate Action Plan", project_action_plan_path(@project), color: "yellow") %>
```

Comment:
I think this action should still be accessible even when archived, the action plan does not modify anything, and I imagine the need of going back to read older action plans to check things